### ISSUE TITLE
[Bug] Missing code coverage in TestWorkflowEnvironment

### ISSUE THREAD

**Issue by joebowbeer** [temporalio/sdk-typescript#729]
### What are you really trying to do?

Observe code coverage of workflow unit tests.

### Describe the bug

All of the workflow lines are reported as being Uncovered even though many are covered by tests.

### Minimal Reproduction

```bash
git checkout https://github.com/temporalio/temporal-pendulum
cd position-node
nvm use
npm i
npm t
```

Note: Because of #728 you may need to run tests twice.

Observe that all of the workflow lines (29-56) are reported as Uncovered even though two signals and a query are tested.

![pass](https://user-images.githubusercontent.com/28679/176032829-d7e1d7be-1f56-4547-9aca-d7cc6312db32.png)

### Environment/Versions

M1 Mac
@temporal/testing 1.0.0-rc.0
Node 16.15.1

**Comment by bergundy** [comment]
Thanks for reporting.
This is probably going to be tricky with the way workflow code executes in the sandbox

**Comment by bergundy** [comment]
Closing since we now have the `@temporalio/nyc-test-coverage` package.
